### PR TITLE
Update project-guidelines-for-stripe.md

### PR DESCRIPTION
**Pull request by stereomon**
## PR Description
Add a meaningful description here that will let us know what you want to fix with this PR or what functionality you want to add.

## Steps before you submit a PR
- Please add tests for the code you add if it's possible.
- Please check out our contribution guide: https://docs.spryker.com/docs/dg/dev/code-contribution-guide.html
- Add a `contribution-license-agreement.txt` file with the following content:
`I hereby agree to Spryker\'s Contribution License Agreement in https://github.com/spryker/spryker-docs/blob/HASH_OF_COMMIT_YOU_ARE_BASING_YOUR_BRANCH_FROM_MASTER_BRANCH/CONTRIBUTING.md.`

This is a mandatory step to make sure you are aware of the license agreement and agree to it. `HASH_OF_COMMIT_YOU_ARE_BASING_YOUR_BRANCH_FROM_MASTER_BRANCH` is a hash of the commit you are basing your branch from the master branch. You can take it from commits list of master branch before you submit a PR.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by matweew — COMMENTED** [review]
For Stripe we know the format of Quote attribute for initial request, why not to add the full structure? Or link to openapi spec.

**Comment by stereomon** [comment]
@matweew GrandTotal is the only field in the QuoteTransfer that is required and gets validated. We don't have a publicly accessible Open API spec available for Stripe.

The format for all PSPs should be the same for the QuoteTransfer.

File: docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ async initializePreOrderPayment() {
       data: {
         type: 'payments',
         attributes: {
-          quote: QUOTE_DATA,
+          quote: QUOTE_DATA, // Must contain the `grandTotal` field and should be as complete as possible
```
Comment (matweew) [inline docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:154]: so where is grandTotal?
 https://github.com/spryker-projects/spa-checkout-glue-with-stripe/blob/main/src/public/assets/main.js#L214-L229

```
    const requestData = {
      data: {
        type: 'payments',
        attributes: {
          quote: {
            customer: this.customer,
            billingAddress: this.address,
            shippingAddress: this.address,
            currency: {
              code: 'EUR',
            },
            store: {
              name: 'DE',
            },
            totals: {
              taxTotal: {
                amount: 100,
              },
            }
          },
          payment: {
            amount: this.grandTotal,
            paymentMethodName: 'stripe',
            paymentProviderName: 'stripe',
          },
          preOrderPaymentData: {
            "transactionId": this.transactionId,
          },
        },
      },
    };
```

Comment (stereomon) [reply docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:154]: Looks like that this is not the problem :( The grandTotal will be added from the payment object to the orderData that gets passed to the App.

Then we have a different problem.

Closing this one.

**Pull request closed without merging**